listenbrainz/webserver/templates/user/music_services.html

```diff
@@ -65,6 +65,40 @@ <h3>A note about permissions</h3>
             </div>
         </div>
 
+        <div class="panel panel-default">
```

Comment:
Suggestions for what to show to users when asking them to link their CB account? I have added a bare bones line the benefit is they can write mini revies. But this should explain it a bit more.

Reply:
Adding onto alastairp's suggestion, we could set it to:

Connect to your CritiqueBrainz account to publish reviews for your **Listens** directly from ListenBrainz. 
Your reviews will be independently visible on CritiqueBrainz and appear publicly on your **CritiqueBrainz profile** unless removed. To view or delete your reviews, visit your **CritiqueBrainz profile.**

Reply:
:+1:

Reply:
Fixed.